### ISSUE TITLE
460.79 driver fails

### ISSUE THREAD

**Issue by daftcoder** [trexminer/T-Rex#105]
Fails on Game Ready Driver 460.79

```
20201209 17:22:39 T-Rex NVIDIA GPU miner v0.18.11  -  [CUDA v11.10]
20201209 17:22:39 r.2c8851280f45
20201209 17:22:39
20201209 17:22:40 WARN: WATCHDOG: T-Rex does not exist anymore, restarting...
20201209 17:22:43 T-Rex NVIDIA GPU miner v0.18.11  -  [CUDA v11.10]
20201209 17:22:43 r.2c8851280f45
20201209 17:22:43
20201209 17:22:43 WARN: WATCHDOG: T-Rex does not exist anymore, restarting...
```

```
20201209 17:23:45 T-Rex NVIDIA GPU miner v0.19.1  -  [CUDA v10.0]
20201209 17:23:45 r.99a7206c3590
20201209 17:23:45
20201209 17:23:46 WARN: WATCHDOG: T-Rex does not exist anymore, restarting...
20201209 17:23:48 T-Rex NVIDIA GPU miner v0.19.1  -  [CUDA v10.0]
20201209 17:23:48 r.99a7206c3590
20201209 17:23:48
20201209 17:23:49 WARN: WATCHDOG: T-Rex does not exist anymore, restarting...
```

**Comment by nils-trubkin** [comment]
Yes! This, same...

**Comment by Stih86** [comment]
+1

**Comment by EvgeniyKorepov** [comment]
+1

**Comment by EvgeniyKorepov** [comment]
What is the problem with the driver version.
NHM3 doesn't work either. More precisely, its device_detection.exe cuda does not work.
https://github.com/nicehash/NiceHashMiner/issues/2234

**Comment by EvgeniyKorepov** [comment]
Looks like NVIDIA released the driver with bugs. Mass Complaints https://www.nvidia.com/en-us/geforce/forums/game-ready-drivers/13/417924/geforce-46079-game-ready-driver-feedback-thread-re/

**Comment by rotor33** [comment]
+1


> Fails on Game Ready Driver 460.79
> 
> ```
> 20201209 17:22:39 T-Rex NVIDIA GPU miner v0.18.11  -  [CUDA v11.10]
> 20201209 17:22:39 r.2c8851280f45
> 20201209 17:22:39
> 20201209 17:22:40 WARN: WATCHDOG: T-Rex does not exist anymore, restarting...
> 20201209 17:22:43 T-Rex NVIDIA GPU miner v0.18.11  -  [CUDA v11.10]
> 20201209 17:22:43 r.2c8851280f45
> 20201209 17:22:43
> 20201209 17:22:43 WARN: WATCHDOG: T-Rex does not exist anymore, restarting...
> ```
> 
> ```
> 20201209 17:23:45 T-Rex NVIDIA GPU miner v0.19.1  -  [CUDA v10.0]
> 20201209 17:23:45 r.99a7206c3590
> 20201209 17:23:45
> 20201209 17:23:46 WARN: WATCHDOG: T-Rex does not exist anymore, restarting...
> 20201209 17:23:48 T-Rex NVIDIA GPU miner v0.19.1  -  [CUDA v10.0]
> 20201209 17:23:48 r.99a7206c3590
> 20201209 17:23:48
> 20201209 17:23:49 WARN: WATCHDOG: T-Rex does not exist anymore, restarting...
> ```

+1

**Comment by trexminer** [comment]
Run the miner with `--no-nvml` as a temporary workaround.

**Comment by trexminer** [comment]
Fixed in 0.19.3
Basically it applies `--no-nvml` automatically. We haven't found a way to keep NVML monitoring working with v460.79

**Comment by rotor33** [comment]
Sorry. Will wait.

**Comment by trexminer** [comment]
Should be fixed in 0.19.4
If not, please let me know.
